src/PrestaShopBundle/Resources/views/Admin/Sell/Order/CreditSlip/Blocks/pdf_by_date.html.twig

```diff
@@ -0,0 +1,61 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% trans_default_domain "Admin.Orderscustomers.Feature" %}
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(pdfByDateForm, {
+  'method': 'GET',
+  'action': path('_admin_credit_slips_pdf_by_date')})
+}}
+<div class="card">
+  <h3 class="card-header">
+    <i class="material-icons">date_range</i> {{ 'By date'|trans }}
+  </h3>
+  <div class="card-block row">
+    <div class="card-text">
+      {% set dateHint = '(Format: 2011-12-31 (inclusive).'|trans({}, 'Admin.Orderscustomers.Help') %}
```

Comment:
It seems there is an extra bracket at the beginning, please write _Format: 2011-12-31 (inclusive)._ instead.